packages/components/src/tour_tip/tour_tip.tsx

```diff
@@ -41,6 +41,7 @@ type Props = {
     zIndex?: number;
     className?: string;
     showBackdrop?: boolean;
+    tippyBlueStyle?: boolean;
```

Comment:
@JulienTant for using the tour tip in the blue style you need to use this parameter as true :)